samples/tab-external-auth/csharp/README.md

```diff
@@ -0,0 +1,122 @@
+---
+page_type: sample
+description: Microsoft Teams sample app which showcases Google authentication using external auth providers.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- csharp
+extensions:
+ contentType: samples
+ createdDate: "16/08/2023 11:20:17 AM"
+urlFragment: officedev-microsoft-teams-samples-tab-external-auth-csharp
+---
+# Tab external auth - C#
+
+This sample illustrates how to implement Google auth using external auth providers.
+
+## Included Features
+* External Auth (Google Oauth2)
+* Tabs
+
+## Interaction with app
+
+![tab-external-auth](Images/tab-external-auth-app.gif)
+
+## Prerequisites
+
+- [.NET Core SDK](https://dotnet.microsoft.com/download) version 6.0
+
+  determine dotnet version
+  ```bash
+  dotnet --version
+  ```
+- [devtunnel](https://aka.ms/TunnelsCliDownload/win-x64) or [ngrok](https://ngrok.com/download) (For local environment testing) latest version (any other tunneling software can also be used)
+
+- [Teams](https://teams.microsoft.com) Microsoft Teams is installed and you have an account
+
+## Setup
+
+1. Create Google Ouath app [Google API Console](https://console.developers.google.com/)
+ - Select Oauth client Id for app creation
```

Comment:
I can't see any documentation to setup external auths.

Reply:
We have added steps inside same readme